logger/sentry.go

```diff
@@ -25,9 +25,10 @@ type SentryLogger struct {
 // routing messages to the DSN provided.
 func NewSentryLogger(env trails.Environment, l Logger, dsn string) Logger {
 	err := sentry.Init(sentry.ClientOptions{
-		Dsn:          dsn,
-		Environment:  env.String(),
-		IgnoreErrors: []string{"write: broken pipe"},
+		Dsn:           dsn,
+		Environment:   env.String(),
+		IgnoreErrors:  []string{"write: broken pipe"},
+		MaxErrorDepth: 2,
```

Comment:
It doesn't appear we wrap more than twice in our projects, so this seems like a sane default in line with Sentry's advice to keep it brief.

I also didn't see a change in exception handling when I wrapped a bunch of errors and changed this number, so 🤷